docs/INSTALL-OSX.md

```diff
@@ -60,20 +60,20 @@ retrieved from a Google Cloud Docker repository.
 The Oak Runtime and its dependencies are built with the following script:
 
 ```bash
-./scripts/build_server
+./scripts/runner build-server
 ```
 
 Build a particular example, say `hello_world`, with:
 
 ```bash
-./scripts/build_example -e hello_world
+./scripts/runner run-examples --example-name=hello_world
```

Comment:
```suggestion
./scripts/runner run-examples --run-server=false --run-clients=false --example-name=hello_world
```

It seems to me this ensures the example is built, but not run.

Reply:
done